packages/types/types/ts3.6/index.d.ts

```diff
@@ -1,4 +1,6 @@
-import { Action } from 'redux'
+export interface Action<T = any> {
+  type: T
+}
```

Comment:
Could you check if this would work? or how much of a butterfly effect this would have on our declared types and implemented type tests?

```suggestion
export interface Action<T extends string = string> {
  type: T
}
```

Reply:
@Andarist It appears to just work (or at least `yarn test:types` seems to succeed in `@redux-saga/core`).